src/acl.c

```diff
@@ -2469,18 +2469,19 @@ void addACLLogEntry(client *c, int reason, int context, int argpos, sds username
     }
 }
 
-const char* getAclErrorMessage(int acl_res) {
-    /* Notice that a variant of this code also exists on aclCommand so
-     * it also need to be updated on changed. */
+sds getAclErrorMessage(int acl_res, user *user, struct redisCommand *cmd, robj **argv, int idx) {
```

Comment:
instead of receiving `argv` and `idx` it's better to send only the key/channel name

Reply:
ok, done